lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
In that example it is necessary because the Group Name is user-provided, and therefore may contain "bad" characters. It's not necessary to do for plain text strings that we define.